deploy/marathon/resources/connected-car.json

```diff
@@ -40,18 +20,17 @@
       "disk": 10,
       "gpus": 0,
       "instances": 2,
-      "backoffSeconds": 1,
-      "backoffFactor": 1.15,
-      "maxLaunchDelaySeconds": 3600,
+      "upgradeStrategy": {
+        "maximumOverCapacity": 0,
+        "minimumHealthCapacity": 0.5
+      },
+      "killSelection": "YOUNGEST_FIRST",
```

Comment:
Works reasonably well for Akka Cluster, since the singleton is usually on the oldest node so it will usually get bounced only once during a deployment.